gradio/blocks.py

```diff
@@ -389,9 +389,7 @@ def serve_static_file(
         else:
             data = {"path": url_or_file_path, "meta": {"_type": "gradio.FileData"}}
             try:
-                return client_utils.synchronize_async(
-                    processing_utils.async_move_files_to_cache, data, self
-                )
+                return processing_utils.move_files_to_cache(data, self)
```

Comment:
THis looks good to me but we need to include the `mark_svg_as_safe` logic to `move_files_to_cache` as well.

Reply:
Thanks, I will fix it.

Reply:
@freddyaboulton Fixed, plz check it 👍